nimbus: Also build nimbus_validator_client

catwith1hat opened the pull request:
This commit make the nimbus package build the stand-alone validator client by default. The stand-alone validator is documented here: https://nimbus.guide/validator-client.html.

selfuryon commented:
Hello! Thanks for the PR!! I see that there are two separate binaries, can you add applications for them too, please? like [here](https://github.com/nix-community/ethereum.nix/blob/main/packages/default.nix#L86)

catwith1hat commented:
I added the two applications to the list. Is it okay that I removed `nimbus.bin` in favor of `nimbus.nimbus_beacon_chain.bin`? I am not using this mechanism, so not sure what's the proper way.

selfuryon commented:
> I added the two applications to the list. Is it okay that I removed `nimbus.bin` in favor of `nimbus.nimbus_beacon_chain.bin`? I am not using this mechanism, so not sure what's the proper way.

Yeah, that's ok!

selfuryon commented on packages/default.nix:
```diff
@@ -97,7 +97,10 @@
       # consensus / nimbus
-      nimbus.bin = "nimbus_beacon_node";
+      nimbus = {
+        nimbus_beacon_node.bin = "nimbus_beacon_node";
```
```suggestion
        nimbus-beacon-node.bin = "nimbus_beacon_node";
```

selfuryon commented on packages/default.nix:
```diff
@@ -97,7 +97,10 @@
-      nimbus.bin = "nimbus_beacon_node";
+      nimbus = {
+        nimbus_beacon_node.bin = "nimbus_beacon_node";
+        nimbus_validator_client.bin = "nimbus_validator_client";
```
```suggestion
        nimbus-validator-client.bin = "nimbus_validator_client";
```

selfuryon reviewed (commented):
Just small fix

selfuryon reviewed (approved):
LGTM!
